docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst

```diff
@@ -47,7 +47,9 @@ This sub-subsection specify the attributes of each particular CE of the site. Mu
 +------------------------------------------------+-------------------------------------------------------------+--------------------------------+
 | *<CE_NAME>/architecture*                       | CE architecture                                             | architecture = x86_64          |
 +------------------------------------------------+-------------------------------------------------------------+--------------------------------+
-| *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                 | CEType = LCG                   |
+| *<CE_NAME>/CEType*                             | Type of CE, can take values as LCG or CREAM                 | CEType = ARC                   |
++------------------------------------------------+-------------------------------------------------------------+--------------------------------+
+| *<CE_NAME>/LocalCEType*                        | Type of Local CE, normally empty. Defaul = "InProcess"      | CEType = Pool                  |
```

Comment:
That should be:
* "`Defaul =`"  --> "`Default = `"  (missing t)
*  "`CEType = Pool`" --> "`LocalCEType = Pool`"